### ISSUE TITLE
CRASH: KUBE_INSECURE=true  causes crash

### ISSUE THREAD

**Issue by kevinkirkup** [hashicorp/terraform-provider-helm#640]
### Terraform Version and Provider Version
```
❯ terraform -v                                                                                                                                                                                                       (ksk/update-to-deploy-airflow|✚1)
Terraform v0.14.0
+ provider registry.terraform.io/hashicorp/helm v1.3.2
+ provider registry.terraform.io/hashicorp/kubernetes v1.13.3
```

### Provider Version
v1.3.2

### Affected Resource(s)
- helm_release

### Terraform Configuration Files
N/A

### Debug Output
N/A. 

NOTE: In addition to Terraform debugging, please set HELM_DEBUG=1 to enable debugging info from helm.-->

### Panic Output
https://gist.github.com/kevinkirkup/0240ac876ed0875ff2d87e91e00d94b7

### Expected Behavior
No Crash

### Actual Behavior
Crash

### Steps to Reproduce

1. `KUBE_INSECURE=true terraform apply`

### Important Factoids
N/A

### References
- https://github.com/hashicorp/terraform-provider-helm/issues/616

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by aareet** [comment]
May be fixed in v2.0.2 of the provider with the SDK upgrade, requires confirmation

**Comment by aareet** [comment]
@kevinkirkup are you able to reproduce this with the latest version of the provider?
